### ISSUE TITLE
Ability to use mulitple .taskcat.yml from scopes

### ISSUE THREAD

**Issue by TejjeT** [aws-quickstart/taskcat#673]
Hello ,

At present taskcat uses .taskcat.yml from a dedicated project. It will be helpful to have the ability to auto-include additional
.taskcat.yml in subfolders in scope.  if this is already possible, could you provide a way for me to do this. This will help me have different sets of tests while dev and ci ( more like integration tests , if I have 100 different tests) , I know we can run a specific tests using `-t` but having a config helps

**Comment by andrew-glenn** [comment]
Interesting use case. Would using `-p subfolder` accomplish what you're looking for?

For what it's worth, while I understand the ask, I don't understand how it would be used in practice. 

Can you elaborate a bit on the pain here ?

**Comment by TejjeT** [comment]
Yes that is exactly what I was referring to , I could separate the tests into the corresponding sub folders , and I could choose to
Run them based on the the current working directory from which I am
Running.

**Comment by andrew-glenn** [comment]
@TejjeT 

The only remaining question I have is around project root. 

```
❯ tree . -a
.
├── .taskcat.yml
├── testA
│   └── .taskcat.yml
├── testB
│   └── .taskcat.yml
├── testC
│   └── .taskcat.yml
└── testD
    └── .taskcat.yml
```

In my mind, each of these tests could be an independent project. Executing `taskcat test run` from the root would collolate each of those tests and run them at once via one execution, rather than independently running taskcat against `testA`, then `testB` etc. 

Does this track with your intended behavior?

**Comment by TejjeT** [comment]
Sorry for super late response, but yes what you described is exactly what I was asking for.

**Comment by sahu-sunil** [comment]
Any updates on this?
I also have similar kind of requirement wherein multiple templates are in different directories, and would like to run them all together by having their own `.taskcat.yml` file.
